### PR TITLE
Rename guardian module path

### PR DESCRIPTION
**Pull request by Lbqds**
Rename the module path to make it available as a dependency library, and it is necessary for #40

### CODE REVIEW

File: node/pkg/proto/node/v1/node.pb.go [hunk 1]
```diff
@@ -0,0 +1,1810 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
```
Comment (h0ngcha0) [inline node/pkg/proto/node/v1/node.pb.go:2]: Do we need to check in all the auto-generated go code?

**Review by h0ngcha0 — COMMENTED** [review]
LGTM, except for one question.

File: node/pkg/proto/node/v1/node.pb.go (hunk 1 above)
Comment (Lbqds) [reply node/pkg/proto/node/v1/node.pb.go:2]: Yes, if other go projects need it, then we don't need to regenerate the code, wormhole also does this: https://github.com/wormhole-foundation/wormhole/tree/dev.v2/node/pkg/proto

**Review by h0ngcha0 — APPROVED** [review]

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**